operations/commit_queue.go

```diff
@@ -213,15 +213,21 @@ func listCommitQueue(ctx context.Context, client client.Communicator, ac *legacy
 	grip.Infof("Queue Length: %d\n", len(cq.Queue))
 	for i, item := range cq.Queue {
 		grip.Infof("%d:", i)
-		author, _ := client.GetCommitQueueItemAuthor(ctx, projectID, restModel.FromStringPtr(item.Issue))
-		if author != "" {
-			grip.Infof("Author: %s", author)
+		issue := restModel.FromStringPtr(item.Issue)
+		p, err := ac.GetPatch(issue)
+		if err != nil {
+			grip.Error(message.WrapErrorf(err, "\terror getting patch for issue '%s'", issue))
```

Comment:
This change has the added benefit of providing the issue ID which would make for easier debugging of this problem.